doc/pages/tutorials/010-astarte_in_5_minutes.md

```diff
@@ -10,6 +10,12 @@ You will need a machine with at least 4GB of RAM, a recent 64-bit operating syst
 
 Also, on the machine(s) or device(s) you will use as a client, you will need either Docker, or a [Qt5](https://www.qt.io/) installation with development components if you wish to build and run components locally.
 
+If on working from a Linux machine, make sure your aio-max-nr is at least 1Mb
```

Comment:
Make it:

Due to ScyllaDB requirements, if you're working on a Linux machine you should make sure that `aio-max-nr` is at least ` 1048576`: [...]